src/extension.ts

```diff
@@ -142,9 +142,11 @@ export async function activate(extensionContext: ExtensionContext): Promise<any>
         OdoImpl.Instance.loadWorkspaceComponents(event);
     });
 
-    OdoImpl.Instance.loadWorkspaceComponents(null);
+    await ComponentTypesView.instance.getAllComponents().then(() => {
+        OdoImpl.Instance.loadWorkspaceComponents(null);
```

Comment:
loadWorkspaceComponents and startTelemetry should not be dependent on loading registry information.